monai/apps/nuclick/transforms.py

```diff
@@ -202,7 +207,7 @@ def __init__(self, keys: KeysCollection, min_size: int = 500, allow_missing_keys
     def __call__(self, data):
         d = dict(data)
         for key in self.keys:
-            img = d[key].array if isinstance(d[key], MetaTensor) else d[key]
+            img = torch.Tensor.numpy(d[key]) if isinstance(d[key], torch.Tensor) else d[key]
```

Comment:
this might not work for cuda tensors:
```py
>>> x = torch.zeros((2,3)).cuda()
>>> torch.Tensor.numpy(x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```

I think this could be replaced by `monai.utils.convert_to_numpy(d[key])`

Reply:
didn't know we have this `monai.utils.convert_to_numpy`... will use the same

Reply:
Fixed